### DISCUSSION TITLE
Where did the types GetFunctionArgs and InferFunctionName go after updating to 2.x.x?

### DISCUSSION THREAD

**Discussion by eldnctlh** [wevm/viem#2099] [Question] [answered]
There is merely no mentions of those types in viem codebase and docs, only some hints in changelog:
![image](https://github.com/wevm/viem/assets/36642337/0415f873-4dad-4682-b0b8-103e411461f1)

**Comment by eldnctlh** [comment]
@jxom Hope for clarification on this

**Reply by jxom** [reply, marked as answer]
It is now `ContractFunctionArgs`.

**Reply by eldnctlh** [reply]
Thanks!
